internal/server/v1/alert/service.go

```diff
@@ -26,6 +26,12 @@ type Service struct {
 	Siren sirenv1beta1grpc.SirenServiceClient
 }
 
+func NewService(siren sirenv1beta1grpc.SirenServiceClient) *Service {
+	return &Service{
+		Siren: siren,
```

Comment:
let's use lowercase `siren` to make it as private variable. We can also name it as `sirenClient`.

Reply:
updated